saleor/csv/utils/export.py

```diff
@@ -10,17 +10,24 @@
 from .. import FileTypes, events
 from ..emails import send_email_with_link_to_download_csv
 from ..models import ExportFile
-from .products_data import get_products_data
+from .products_data import get_export_fields_and_headers_info, get_products_data
 
 if TYPE_CHECKING:
     # flake8: noqa
     from django.db.models import QuerySet
 
 
+BATCH_SIZE = 10000
```

Comment:
I wonder if that's the right place? I'm not really a fan of tunable values being inside python (code) files. Seems like with meaningful name it would be better of inside `settings.py`? Unless there is good a reason.

Reply:
But it concerns only export

Reply:
But should this value be configurable? It's rather a constant where we should find the optimal value in tests on different datasets.